README.md

```diff
@@ -1,5 +1,208 @@
+<img align="center" src="./dosc/media/../../docs/media/api-governance-processes.png" width="600">
+</div>
+  
+<p>&nbsp;</p>
+
+### Installation Guide
+
+#### Registry services
+This will start the registry components, `API specification registry` and `Service Registry`.
+
+1. Fetch the demonstration github repository.   
+    ```sh
+    git clone git@github.com:Sinetiq/api-specification-governance.git
+    ```
+
+2. Change directory to the root git folder where the docker-compose.yml can be found, run the docker command.
+    ```sh
+    cd api-specification-governance
+
+    docker-compose up
+    ```
+
+_You can stop your registry services with CTRL-C._
+
+#### Service explorer
+Download and configure Visual Studio Code as your demonstration editor. Install the SINETIQ Service Explorer plugin. Follow the readme at the visual studio code marketplace [https://marketplace.visualstudio.com/items?itemName=Sinetiq.service-explore](https://marketplace.visualstudio.com/items?itemName=Sinetiq.service-explore).
+
+#### CLI tool: Manual run (not devops pipeline)
+
+1. Download CLI tool component from [https://www.npmjs.com/package/@sinetiq/api-specification-registry-cli](https://www.npmjs.com/package/@sinetiq/api-specification-registry-cli). Configure the CLI for your GIT repository containing API specifications (YAML files).
+   
+```sh
+npm i @sinetiq/api-specification-registry-cli
+```
+
+#### CLI tool: Automation, devops (CI/CD) pipeline
+
+1. Use `pipeline example` matching your environment.
+
+2. Create the API specification and version control it in your git-repo, attached to your created pipeline.
+
+3. Create a Pull Request/Merge Request, review, approve and merge the API specification, initiate the pipeline.
+
+
+_Your demonstration environment is now ready._
+
+#### Verify installation
+
+1. Verify the API Specification Registry.  
+   Open a webbrowser, use url http://localhost:3333/ to verify up-n-running.
+
+2. Verify Service Registry.  
+   Open a webbroser, use url http://localhost:8500/ to verify up-n-running.
+
+3. Verify Service Explorer.  
+   Open Visual Code, verify the `Service Explroer` accroding to user manual.
+
+4. Verify CLI-installed.
+   ```sh
+   npm -v @sinetiq/api-specification-registry-cli
+   ```
+
+<p>&nbsp;</p>
+
+## Usage
+
+The steps 1 to 5 will show how to run this demonstration and successfully mange `governance of API specifications` in the complete life cycle chain.
+
+<div style="text-align:center">
+<img align="center" src="./dosc/media/../../docs/media/api-governance-processes-steps.png" width="600">
+</div>
+<p>&nbsp;</p>
+
+### Step 1: Create API Specification
+Use your preferred editor to create a YAML file for the API specification and store it in your versioning repository (ex. git). Ensure each specification has an API identifier (`x-api-identifier`) with a unique name. _The API specifications will be loosely coupled, no server part and only one specific information object (resource)._ 
```

Comment:
We have not discussed the `x-api-identifier`. We should remove it for now and discuss it. If we are going to embed something into the spec it should rather be the coordinates (group, artifact and version).

Regarding one resource, I understand what you want to accomplish but this is overly restrictive. First, it assumes only REST is used. Second, a service can have different resources even though it is scoped to a reasonable size. Different resources are often used to map relationships from different directions/perspectives.

Reply:
I think I was more into Information Object, and only one "part of information" should be handled within one api specification. However, I think we need to discuss this as you mention.